cluster/images/etcd/rollback/rollback.go

```diff
@@ -322,12 +322,14 @@ func toTTLOptions(r *pb.Request) store.TTLOptionSet {
 }
 
 func applyRequest(r *pb.Request, applyV2 etcdserver.ApplierV2) {
+	// TODO: find a sane way to perform this cast or avoid it in the first place
+	reqV2 := &etcdserver.RequestV2{r.ID, r.Method, r.Path, r.Val, r.Dir, r.PrevValue, r.PrevIndex, r.PrevExist, r.Expiration, r.Wait, r.Since, r.Recursive, r.Sorted, r.Quorum, r.Time, r.Stream, r.Refresh, r.XXX_unrecognized}
```

Comment:
I can see that this change is because of PR #9826, which was implemented by yourself, so I guess you knew you need to make this fix. Then if someone else made some changes that you are not aware of or not remember, when upgrading, do we just have to do something like this and try out if test fails? For example, the commit of "Fix tests for changes in etcd 3.3.0-rc.3" is because of PR #7909.